### PR TITLE
Use all abilities near end game

### PR DESCRIPTION
**Pull request by DanielMcAssey**
Fix for #226, use all abilities at end game.

### CODE REVIEW

File: autoPlay.user.js [hunk 1]
```diff
@@ -408,8 +408,12 @@ function MainLoop() {
 		goToLaneWithBestTarget(level);
 
 		attemptRespawn();
-
-		useAbilities(level);
+		
+		if(isNearEndGame()) {
```
Comment (xPaw) [inline autoPlay.user.js]: `isNearEndGame` will return true in last hour, but I want it to do that only in last 15 minutes.
Modify `isNearEndGame` to return minutes left until the game ends, and do a check based on that.

File: autoPlay.user.js [hunk 2]
```diff
@@ -495,6 +499,14 @@ function MainLoop() {
 	}
 }
 
+function useAllAbilities() {
+	for(var key in ABILITIES) {
+		if(canUseAbility(ABILITIES[key])) {
```
Comment (xPaw) [inline autoPlay.user.js]: Ignore wormholes here.

Comment (xPaw) [inline autoPlay.user.js]: Oh yea, you don't even need to call `canUseAbility`, `tryUsingAbility` already performs these checks.

**Comment by DanielMcAssey** [comment]
Messed up rebase. Sorry! :(

**Pull request closed without merging**